board/safety/safety_mazda.h

```diff
@@ -22,7 +24,8 @@
 #define MAZDA_DRIVER_TORQUE_ALLOWANCE 15
 #define MAZDA_DRIVER_TORQUE_FACTOR 1
 
-int mazda_cruise_engaged_last = 0;
+const AddrBus MAZDA_TX_MSGS[] = {{MAZDA_LKAS, 0}};
+
 int mazda_rt_torque_last = 0;
 int mazda_desired_torque_last = 0;
 uint32_t mazda_ts_last = 0;
```

Comment:
This was refactored recently, can you update this? should be `rt_torque_last`, `ts_last`, etc.